api/src/main/java/io/strimzi/api/kafka/model/KafkaMirrorMaker2.java

```diff
@@ -0,0 +1,167 @@
+                },
+                subresources = @Crd.Spec.Subresources(
+                        status = @Crd.Spec.Subresources.Status()
+                ),
+                additionalPrinterColumns = {
+                        @Crd.Spec.AdditionalPrinterColumn(
+                                name = "Desired replicas",
+                                description = "The desired number of Kafka Mirror Maker 2 replicas",
+                                jsonPath = ".spec.replicas",
+                                type = "integer"
+                        )
+                }
+        )
+)
+@Buildable(
+        editableEnabled = false, 
+        generateBuilderPackage = false, 
+        builderPackage = "io.fabric8.kubernetes.api.builder", 
+        inline = @Inline(type = Doneable.class, prefix = "Doneable", value = "done")
+)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({ "apiVersion", "kind", "metadata", "spec", "status" })
+@EqualsAndHashCode
+public class KafkaMirrorMaker2 extends CustomResource implements UnknownPropertyPreserving, HasStatus<KafkaMirrorMaker2Status> {
+
+    private static final long serialVersionUID = 1L;
+
+    public static final String SCOPE = "Namespaced";
+    public static final String V1ALPHA1 = "v1alpha1";
+    public static final String V1BETA1 = "v1beta1";
+    public static final List<String> VERSIONS = unmodifiableList(asList(V1BETA1, V1ALPHA1));
+    public static final String RESOURCE_KIND = "KafkaMirrorMaker2";
+    public static final String RESOURCE_LIST_KIND = RESOURCE_KIND + "List";
+    public static final String RESOURCE_GROUP = "kafka.strimzi.io";
+    public static final String RESOURCE_PLURAL = "kafkamirrormaker2s";
+    public static final String RESOURCE_SINGULAR = "kafkamirrormaker2";
+    public static final String CRD_API_VERSION = "apiextensions.k8s.io/v1beta1";
+    public static final String CRD_NAME = RESOURCE_PLURAL + "." + RESOURCE_GROUP;
+    public static final String SHORT_NAME = "kmm2";
+    public static final List<String> RESOURCE_SHORTNAMES = singletonList(SHORT_NAME);
+
+    private String apiVersion;
+    private KafkaMirrorMaker2Spec spec;
+    private ObjectMeta metadata;
+    private KafkaMirrorMaker2Status status;
+    private Map<String, Object> additionalProperties = new HashMap<>(0);
+
+    @Override
+    public String getApiVersion() {
+        return apiVersion;
+    }
+
+    @Override
+    public void setApiVersion(String apiVersion) {
+        this.apiVersion = apiVersion;
+    }
+
+    @JsonProperty("kind")
+    @Override
+    public String getKind() {
+        return RESOURCE_KIND;
+    }
+
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
+    @Override
+    public ObjectMeta getMetadata() {
+        return super.getMetadata();
+    }
+
+    @Override
+    public void setMetadata(ObjectMeta metadata) {
+        super.setMetadata(metadata);
+    }
+
+    @Description("The specification of the Kafka Mirror Maker 2 cluster.")
```

Comment:
Isn't it normally spelt "MirrorMaker"?

Reply:
Yep - I've updated all places where I used `Mirror Maker 2` to be `MirrorMaker 2.0`, in line with Kafka's naming.